frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx

```diff
@@ -302,12 +343,13 @@ export default class LineAreaBarChart extends Component {
         )}
       >
         {hasTitle && (
-          <ChartCaption
-            series={series}
-            settings={settings}
+          <LegendCaption
+            className="mx1 flex-no-shrink"
```

Comment:
Please consider making this a styled component.

Ideally we phase out these class names with styles.